package.json

```diff
@@ -1,46 +1,45 @@
 {
-  "name": "@condenast/quick-bus",
-  "version": "0.2.0",
-  "description": "A tiny and fast event bus with AMQP and Postal.js-like functionality",
-  "main": "index.min.js",
-  "files": [
-    "index.min.js"
-  ],
-  "browserslist": [
-    "last 2 years"
-  ],
-  "scripts": {
-    "build": "uglifyjs --compress --mangle --output index.min.js -- index.js",
-    "lint": "eslint --report-unused-disable-directives .",
-    "test": "npm run karma-once",
-    "posttest": "eslint --report-unused-disable-directives .",
-    "karma": "karma start",
-    "karma-once": "karma start --single-run",
-    "prepublish": "npm test -- --single-run && npm run build"
-  },
-  "keywords": [
-    "postal",
-    "amqp",
-    "event bus",
-    "event broker",
-    "broker",
-    "events"
-  ],
-  "author": "Dane Stuckel",
-  "license": "Apache-2.0",
-  "dependencies": {},
-  "devDependencies": {
-    "chai": "^4.2.0",
-    "eslint": "^6.1.0",
-    "eslint-plugin-compat": "^3.3.0",
-    "eslint-plugin-import": "^2.18.2",
-    "karma": "^4.2.0",
-    "karma-chai": "^0.1.0",
-    "karma-mocha": "^1.3.0",
-    "karma-phantomjs-launcher": "^1.0.4",
-    "karma-sinon": "^1.0.5",
-    "mocha": "^6.2.0",
-    "sinon": "^7.3.2",
-    "uglify-js": "^3.6.0"
-  }
+    "name": "@condenast/quick-bus",
+    "version": "1.0.0",
```

Comment:
Can we do 0.3.0 since we haven't hit 1.0 before? We'll need to manually publish this after it's merged.

Also, I think you have your code editor set to 4 spaces, it should really be 2 spaces otherwise npm will be weird next time it touches the file.

Reply:
thinking about this. Since the client code does not change, do we really need to publish? This is basically now a DevX ticket?

Reply:
Rollback version bump?